packages/theme-data/src/baseTheme/system/density.js

```diff
@@ -1,6 +1,7 @@
 import { FONT_SIZE, SPACING } from "../../consts/types";
 
 export default {
+  "spacings.none": { type: SPACING, value: { ref: "basics.spacings.none" } },
```

Comment:
from looking at the rest of the objects in this file i don't think `value` should go here, but instead into the `densidties/highOrMediumDensity/sytem/density.js` files. I think we are mainly only setting `type` here.

Reply:
Yah I know what you mean and I noticed that too, but I still think we should move it as `colorScheme` is like that where even themes with roles of the same values are still in each theme's `colorScheme`.

Reply:
actually thinking about it some more, if it's the same across, maybe we don't even need to put it into the `system` and just reference it through `basics.spacings.none`.

Reply:
it makes sense for the other spacings since they do change in size based on density, but i don't see `none` ever changing, so might be better to just leave it out.